src/alexandra_ai_data/domsdatabasen.py

```diff
@@ -0,0 +1,90 @@
+"""API for accessing processed data from Domsdatabasen."""
+
+from omegaconf import DictConfig
+
+from domsdatabasen import DatasetBuilder, Scraper, Processor
+
+from .config_domsdatabasen import config
+
+from logging import getLogger
+
+from datasets import load_dataset
+
+logger = getLogger(__name__)
+
+
+class Domsdatabasen:
+    """API for accessing processed data from Domsdatabasen
+
+    Attributes:
+        config (DictConfig):
+            Configuration settings object.
+        scraper (Scraper):
+            Scraper object for scraping data from Domsdatabasen.
+        processor (Processor):
+            Processor object for processing scraped data.
+        dataset_builder (DatasetBuilder):
+            DatasetBuilder object for building dataset samples.
+        dataset (Dataset):
+            Dataset of processed data from Domsdatabasen.
+    """
+
+    def __init__(self):
+        """Initialize."""
+        self.config: DictConfig = config
+        self.dataset = load_dataset("alexandrainst/domsdatabasen")["train"]
```

Comment:
Consider using the `split` argument instead:

```suggestion
        self.dataset = load_dataset("alexandrainst/domsdatabasen", split="train")
```